chainerx_cc/chainerx/routines/indexing.h

```diff
@@ -27,19 +42,7 @@ Array At(const Array& a, const std::vector<ArrayIndex>& indices);
 // `axis` must be within [0, b.ndim()).
 // `indices` must have dtype kind of either kInt or kUInt.
 
-class AddAtOp : public Op {
-public:
-    static const char* name() { return "AddAt"; }
-
-    virtual void Call(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out);
-
-protected:
-    virtual void Impl(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out) = 0;
-};
-
-inline void AddAt(const Array& a, const Array& indices, int8_t axis, const Array& b, const Array& out) {
```

Comment:
For the record (was told by @asi1024 offline) that the signature of the `AddAt` routine was previously modified to accept the output as an argument opposed to returning it as the interface accepting the output as argument was necessary for internal use, e.g. pooling. However, this PR now gets around the problem since the original device method-like signature of `AddAt` is reintroduced.